tensorflow/lite/tools/optimize/operator_property.cc

```diff
@@ -106,6 +106,12 @@ OperatorProperty GetOperatorProperty(OpVariant op_variant) {
       property.version = 2;
       property.quantizable_int16 = false;
       break;
+    case BuiltinOperator_BROADCAST_TO:
+      property.inputs = {{0, {}}};
+      property.outputs = {{0, {}}};
+      property.restrict_same_input_output_scale = true;
+      property.version = 3;
```

Comment:
Incremented version (and added corresponding op_version tests) since the input/output types of the operator changed, even though the kernel was unchanged.